packages/react-liveness/src/components/FaceLivenessDetector/service/utils/createRequestStreamGenerator/utils.ts

```diff
@@ -95,7 +105,36 @@ const getTargetFaceBoundingBox = (
   });
 };
 
+type ClientChallenge =
+  | FaceMovementAndLightClientChallenge
+  | FaceMovementClientChallenge;
+interface CreateClientSessionInformationEventParams {
+  sessionInformation: SessionInformation;
+  clientChallenge: ClientChallenge;
+}
+
+function createClientSessionInformationEvent({
```

Comment:
Basically just pulling out the type checking/formatting of return type into its own util